Makefile

```diff
@@ -14,40 +14,83 @@ PROJECT_DIR := $(realpath $(shell dirname $(firstword $(MAKEFILE_LIST))))
 
 BIN_PATH=$(PROJECT_DIR)/bin
 
-.PHONY: install install_kjob install_kueuectl install_gcloud check_python update-path 
-
+.PHONY: install
 install: check-python check-gcloud install-kueuectl install-kjob pip-install
 
-install-dev: check-python check-gcloud mkdir-bin install-kubectl install-kueuectl install-kjob pip-install install-pytest 
+.PHONY: install-dev
+install-dev: check-python check-gcloud mkdir-bin install-kubectl install-kueuectl install-kjob pip-install install-pytest
 
+.PHONY: pip-install
```

Comment:
It looks like for targets we're following the pattern `install-[dep_name]`. Can we rename this to `install-pip`?

Reply:
With pip-install we are installing dependencies but not a pip.